content/en/docs/ops/configuration/security/security-policy-patterns/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Security Policy Patterns
+description: Shows common patterns of using Istio security policy.
+weight: 60
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+## Background
+
+This page shows common patterns of using Istio security policy. You may find them useful in your deployment or use this
+as a quick reference to example policies.
+
+The policy demonstrated here are just examples and please make necessary changes to adapt to your actual environment
+before applying.
+
+Also read the [authentication tasks](/docs/tasks/security/authentication/authn-policy) and the
+[authorization tasks](/docs/tasks/security/authorization) for hands-on tutorial of using the security policy with much
+more details.
+
+### Require mandatory JWT validation
+
+The `RequestAuthentication` policy defines a list of JWT issuers that are allowed, by default it also allows
+requests without JWT token.
+
+After defining the `RequestAuthentication` policy, use the following extra policy to enforce mandatory JWT validation
+that rejects the request if it has no JWT token:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: require-jwt
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
+EOF
+{{< /text >}}
+
+### Require different JWT issuer per host
+
+JWT validation is commonly used on the ingress gateway and you may want to enforce different JWT issuers on different
+hosts, use the following extra policy in addition to the request authentication policy:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # assuming the JWT token issued for example.com has the issuer example.com
+        notRequestPrincipals: ["*@example.com"]
```

Comment:
Why is it in the "notRequestPrincipals" field? Can you just specify expected "requestPrincipals"?

Reply:
this is due to the use of `DENY` policy, this is to achieve the effect that the access control is only applied to the `example.com` and `another.org` hosts but not any other hosts.

If we use requestPrincipals, we need first change the action to `ALLOW` and add a 3rd rule to allow hosts other than `example.com` and `another.org`.

I updated to include example for both use cases.